### DISCUSSION TITLE
updating smtp config

### DISCUSSION THREAD

**Discussion by vengefulpunk** [mastodon/mastodon#25994] [Q&A] [answered]
I feel stupid asking this, but I could not find in any of the documents(which appears to be mainly targeted at new installations and not updating a current installation) how to update the smtp information. I assume it is in a config file or could possibly have a command to update it. 

Thanks!

**Comment by StrangeMinds1960** [comment]
It should be in /home/mastodon/live/.env.production

**Comment by justdaj** [comment, marked as answer]
It is in your mastodon home folder under live/.env.production  

Once your edit it, restart mastodon

 `systemctl reload mastodon-web`

**Comment by vengefulpunk** [comment]
thank you both very much
